docs/source/concepts/local-ledger.rst

```diff
@@ -343,8 +344,14 @@ Local ledgers
 As explained in the Daml Ledger Model, parties see only a :ref:`projection <da-model-projections>` of the shared ledger for privacy reasons.
 Like consistency, projection extends to causality graphs as follows.
 
+Definition »Stakeholder informee«
+  A party `P` is a **stakeholder informee** of an action `act` if `P` is an informee of `act` and, if `act` is an action on a contract then `P` is a stakeholder of the contract.
```

Comment:
I found this a bit tricky to read, I think using bullet points would help

Reply:
Changed to use bullet points.